Source/WebKit/UIProcess/API/wpe/WPEView.cpp

```diff
@@ -319,6 +319,9 @@ View::View(struct wpe_view_backend* backend, const API::PageConfiguration& baseC
 
 View::~View()
 {
+    wpe_view_backend_set_backend_client(m_backend, nullptr, nullptr);
+    wpe_view_backend_set_input_client(m_backend, nullptr, nullptr);
```

Comment:
You can do something like:

```suggestion
    wpe_view_backend_set_backend_client(m_backend, nullptr, nullptr);
    wpe_view_backend_set_input_client(m_backend, nullptr, nullptr);
    // Although the fullscreen client is used for libwpe 1.11.1 and newer, we cannot
    // unregister it prior to 1.15.2 due to <link to libwpe bug here>
#if ENABLE(FULLSCREEN_API) && WPE_CHECK_VERSION(1, 15, 2)
    wpe_view_backend_set_fullscreen_client(m_backend, nullptr, nullptr);
#endif
```

This way we get the code correct in WebKit, but without introducing crashes for current versions of libwpe. (We'll need to make sure that assertion really does get removed in the next release.)

Reply:
I'll wait for feedback on [the PR ](https://github.com/WebPlatformForEmbedded/libwpe/pull/129) and update this code if that patch is merged.

Reply:
Update this PR with suggested changes. PTAL.